firmware/controllers/engine_cycle/main_trigger_callback.cpp

```diff
@@ -130,33 +124,24 @@ void turnInjectionPinHigh(InjectionEvent *event) {
 		InjectorOutputPin *output = event->outputs[i];
 
 		if (output) {
-			turnInjectionPinHigh(output);
+			output->open();
 		}
 	}
 }
 
-static inline void turnInjectionPinLow(InjectorOutputPin *output) {
+void InjectorOutputPin::close() {
 #if FUEL_MATH_EXTREME_LOGGING
-	printf("turnInjectionPinLow %s %d %d\r\n", output->name, output->overlappingCounter, (int)getTimeNowUs());
+	printf("turnInjectionPinLow %s %d %d\r\n", name, overlappingCounter, (int)getTimeNowUs());
 #endif /* FUEL_MATH_EXTREME_LOGGING */
 
-
+	overlappingCounter--;
```

Comment:
this line is added but no matching line removed? is this a feature or a bug?

Reply:
look at line 151 in the "before"

Reply:
fixing the indentation confused git about which line this was :)